internal/controllers/analyzer/analyzer.go

```diff
@@ -159,30 +161,45 @@ func (a *Analyzer) formatAnalysisToSendToAPI() {
 	}
 }
 
-// nolint:gocyclo
-func (a *Analyzer) checkIfNoExistHashAndLog(list []string) {
-	for _, hash := range list {
-		existing := false
-		for idx := range a.analysis.AnalysisVulnerabilities {
-			vulnHash := a.analysis.AnalysisVulnerabilities[idx].Vulnerability.VulnHash
-			vulnHashInvalid := a.analysis.AnalysisVulnerabilities[idx].Vulnerability.VulnHashInvalid
-			if hash == vulnHash || hash == vulnHashInvalid {
-				existing = true
+// logWarnIfHashDontExistsInConfig logs a warning if the one of the config hashes don't exist in the analysis.
+// nolint
+func (a *Analyzer) logWarnIfHashDontExistsInConfig(configHashes []string) {
+	for _, configHash := range configHashes {
+		isDontExists := true
```

Comment:
I don't think I understand the suggestion.

Reply:
change isDontExists to exists,

Reply:
But the function only logs if it does not exist, it will stop making sense.

Reply:
for me a variable named `isDontExists` is a duplicate negative because is just a negation of an `exists` variable, we'll have to invert the if logics to make the rest of the code right but it will be more readable
```go
exists := false
isDontExists := true
fmt.Println(!exists == isDontExists) // will print true
```
do you agree `isDontExists` make the code more confusing?

Reply:
I was thinking it was related to the function name, with the example it became more clear what you meant.

Reply:
Done!